src/ja/server/diagrams/schedulerLowLevel.puml

```diff
@@ -0,0 +1,129 @@
+@startuml
+
+box "Central Server" #Lightblue
+    participant ":QueuingActivity"
+    participant ":Scheduler"
+    participant "job:Job"
+    participant ":RunningJobEntry"
+    participant ":DataBaseHandler"
+    participant ":Dispatcher"
+end box
+
+activate "job:Job"
+-> ":QueuingActivity":
+activate ":QueuingActivity"
+
+":QueuingActivity" -> ":Scheduler" ++: queueJob(job:Job)
+
+note right of ":QueuingActivity"
+  user calls 
+  "add" command
+end note
+
+":Scheduler" -> "job:Job" ++: getPriority(job)
+"job:Job" --> ":Scheduler" --: p: Priority
+
+":Scheduler" -> ":Scheduler" ++: findWC(job)
+":Scheduler" -> ":DataBaseHandler" ++: getWCList()
+note left of ":Scheduler"
+    Scheduler decides which jobs
+    should run on each machine
+end note
+":DataBaseHandler" --> ":Scheduler" --: WCList:list[]
+":Scheduler" --> ":Scheduler" --: wmID:int
+
+":Scheduler" -> "job:Job" ++: set_wm_ID(wmID)
+"job:Job" --> ":Scheduler" --:
```

Comment:
Replace lines 26 to 36 with the following:
* Retrieve list of Jobs from Database
* Retrieve list of WCs from Database
* Sort Jobs by priority

For each Job:
* find_wc
* Set Job wm_id